docs/source/glossary.rst

```diff
@@ -3,6 +3,39 @@ Glossary
 
 .. glossary::
 
+  Block
+    Part of the `blockchain <https://en.wikipedia.org/wiki/Block chain>`__
+    that contains one or more operations (called `transactions`) and a link to
+    the previous block.
+
+  Client
+    Machine that sends requests to and receives replies from the network of
+    Sawtooth nodes. PBFT has no direct interaction with clients; the validator
+    bundles all client requests into blocks and sends them through the
+    consensus API to the consensus engine.
+
+  Consensus API
+    Sawtooth component that abstracts consensus-related interactions between
+    the validator and a consensus engine. The consensus API allows `dynamic
+    consensus`, a feature that allows a choice of consensus for a Sawtooth
+    network.
+
+  Consensus engine
+    Component that provides consensus functionality for a Sawtooth
+    network. The consensus engine communicates with the validator through
+    the consensus API.
+
+  Consensus seal
+    Proof that a block went through consensus.
+
+  Member node
+    Sawtooth node that participates in PBFT consensus. Membership is controlled
+    by the on-chain setting ``sawtooth.consensus.pbft.members``.
```

Comment:
Might be worth adding a sentence like "Every member node is either a :term:`primary node` or a :term:`secondary node`."

Reply:
Added.